python/style.rst

```diff
@@ -562,13 +541,12 @@ Within a module, follow the order:
 1. Shebang line, ``#! /usr/bin/env python`` (only for executable scripts)
 2. Module-level comments (such as the `license statement <https://github.com/lsst/templates/tree/master/file_templates/stack_license_py>`__)
 3. Module-level docstring
-4. ``from __future__ import absolute_import, division, print_function``
-5. ``__all__ = [...]`` statement, if present
-6. Imports (except ``from __future__ import...``)
-7. Private module variables (names start with underscore)
-8. Private module functions and classes (names start with underscore)
-9. Public module variables
-10. Public functions and classes
+4. ``__all__ = [...]`` statement, if present
```

Comment:
Not that it's a big deal, but is there an ReST way to do this so we don't have to manually renumber?